example_data/sample_bundle.json

```diff
@@ -528,21 +634,37 @@
         "relatedArtifact": [
           {
             "type": "citation",
-            "citation": "(1) Shapiro et al Pediatr Res. 2010 Nov;68(5):446-51. (2) Neul et al Am J Med Genet B Neuropsychiatr Genet. 2019 Jan;180(1):55-67. (3) Cheadle et al, Hum Mol Genet. 2000, 9, 1119-1129.",
-            "display": "Gene reference"
+            "citation": "Zheng-Wen et al (2014) Int J Mol Sci. Dec 16;15(12):23408-17; [2] Lee et al (2012) PLoS One. 7(8); [3] M\u00c3\u00a9neret et al (2012) Neurology. Jul  10;79(2):170-4."
+          }
+        ],
+        "action": [
+          {
+            "prefix": "1",
+            "description": "Screening of 82 genes associated with severe delay and seizures () was carried out using next generation sequencing (Agilent SureSelect + MiSeq/NextSeq). *Pseudogenes with significant sequence homology to DMN1 exons 18, 19, 20 and 22 may result in incorrect coverage data for this gene and variants in these exons may not be detected. 99.3% of the coding bases in the targeted genes were covered >30x. Sequencing includes the coding region of the genes targeted, including intron/exon splice regions (-14 splice acceptor and +6 splice donor, unless otherwise specified). Inhouse validation attributes a minimum sensitivity of 99.9% to detect single nucleotide substitutions (with 95% confidence) and 77% for insertion/deletion variants (with 95% confidence) for regions covered by 30 or more reads. Variants are classified using the ACMG/AMP guidelines (Richards et al 2015 Genet Med) /ACGS Best Practice guidelines (2020).",
+            "title": "Test Method"
```

Comment:
before we had this in the `description` field, can you confirm which we should be using from now on?

Reply:
intentional change